doc/controllers/subscription-components.md

```diff
@@ -177,7 +177,7 @@ try {
       "created_at": "2022-02-22T14:07:00-05:00",
       "updated_at": "2022-02-22T14:07:00-05:00",
       "component_handle": "string",
-      "archived_at": "string"
```

Comment:
Something is wrong here, shouldn't it be a date?

Reply:
I've changed the type to date, and put null in the example, as it's more common to have it as null in the response